### PR TITLE
Stop showing ETA after download is complete

### PR DESCRIPTION
**Pull request by djc**
Alternative to #3826 (cc @barafael).

### CODE REVIEW

File: src/cli/download_tracker.rs [hunk 1]
```diff
@@ -220,6 +220,17 @@ impl DownloadTracker {
     }
 }
 
+struct Eta(Duration);
+
+impl fmt::Display for Eta {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self.0 {
+            Duration::ZERO => Ok(()),
+            _ => f.write_fmt(format_args!("{}", self.0.display())),
```
Comment (rami3l) [inline src/cli/download_tracker.rs]: Did you mean something like...

```suggestion
            _ => write!(f, " ETA: {}", self.0.display()),
```

**Review by rami3l — APPROVED** [review]

File: src/cli/download_tracker.rs (hunk 1 above)
Comment (djc) [reply src/cli/download_tracker.rs]: Yup, that's better!

**Comment by rami3l** [comment]
@djc Just tried the latest build and this looks incorrect.

image

**Comment by djc** [comment]
:( Can you file an issue or PR?

**Pull request merged**